### PR TITLE
#864edj4g7 improve vsdia ventura

### PR DESCRIPTION
**Pull request by viniciusventura29**
##### Description of changes

- [X] Why (Why did you make this pull request): 
- To improve css
- Fix bugs 
- Add functionalities to VSDiA project index

- [X] Brief description of the changes: 
- Fix Invalidate queries functionalities
- Improve css in vsdia project index ([id].tsx)
- Improving actionBox Component (adding position and changing css)
- Improving css of CreateLane component
- Fixing sideBarCardInfo component
- Fix circleUserButton icon size

- [X] Possible breakouts (Possible places where things could break): I hope any but there is some functionalities that we have to do in future. User should choose the lane color and when user try to delete lane with cards inside, it should show a alert confirming that decision.

### CODE REVIEW

File: apps/web-vsdia/src/components/ActionBox.tsx [hunk 1]
```diff
@@ -1,73 +1,105 @@
...
   const queryClient = useQueryClient();
-  const createCardMutation = useMutation(['createCard'], api.laneRouter.createCard.mutate, {
-    onSuccess: (_data) => {
-      queryClient.invalidateQueries(['getLanes'])
+  const createCardMutation = useMutation(
+    ["createCard"],
+    api.laneRouter.createCard.mutate,
+    {
+      onSuccess: async (_data) => {
+        await queryClient.invalidateQueries(["getLanes"]);
```
Comment (viktormarinho) [inline apps/web-vsdia/src/components/ActionBox.tsx:28]: I think now we need the idVsdia here too

File: apps/web-vsdia/src/components/ActionBox.tsx [hunk 2]
```diff
@@ -1,73 +1,105 @@
...
     return (
-      <div className="w-80 h-48 bg-gray-100 rounded-lg ml-24" onClick={() => setSelectedCard(card)}>
+      <div
+        className="w-80 h-full bg-white rounded-lg ml-24 cursor-grabbing"
+        onClick={() => setSelectedCard(card)}
+      >
         <div className=" border-b border-black flex flex-row justify-between">
           <div className="ml-2 mt-2 mb-2">
-            {id ?? "0"}
+          {position !== undefined? position : ""}
```
Comment (viktormarinho) [inline apps/web-vsdia/src/components/ActionBox.tsx:66]: position ?? ""

File: apps/web-vsdia/src/components/Droppable.tsx [hunk 3]
```diff
@@ -25,21 +26,32 @@ export const Lane = ({ items, id, name }: LaneProps) => {
             await queryClient.invalidateQueries(['getLanes']);
```
Comment (viktormarinho) [inline apps/web-vsdia/src/components/Droppable.tsx:26]: i think we need the vsdia Id here as well.

File: apps/web-vsdia/src/components/Droppable.tsx [hunk 4]
```diff
@@ -25,21 +26,32 @@ export const Lane = ({ items, id, name }: LaneProps) => {
...
         }
     })
+    const deleteLaneMutation = useMutation(['deleteLane'], api.laneRouter.deleteLane.mutate,{
+      onSuccess: async()=>{
+        await queryClient.invalidateQueries(['getLanes'])
```
Comment (viktormarinho) [inline apps/web-vsdia/src/components/Droppable.tsx:31]: and here too

**Review by viktormarinho — CHANGES_REQUESTED** [review]
Looking nice! just need to change this simple detail - i think the invalidateQueries is either not working or invalidating more than it needs to - then we can merge it

**Review by viktormarinho — APPROVED** [review]
Excelente 😋😋😋

**Comment by vitormiura** [comment]
> HSAUSHAUSAHUX EU SOU O CAMUDO SAHSUAHUHSUHSA LOL 😝 😝 😝 😝 😝

?

**Comment by viniciusventura29** [comment]
> HSAUSHAUSAHUX EU SOU O CAMUDO SAHSUAHUHSUHSA LOL 😝 😝 😝 😝 😝

Que isso cara?

**Review by vitormiura — APPROVED** [review]
looks good to me 👍

**Pull request merged**